syscall_fixed.go

```diff
@@ -0,0 +1,5 @@
+// +build plan9 windows
```

Comment:
We probably also want this for `js,wasm`:

 https://github.com/golang/go/blob/master/src/syscall/syscall_js.go#L159

Reply:
done.